packages/models-library/src/models_library/docker.py

```diff
@@ -143,7 +143,7 @@ def to_simcore_runtime_docker_labels(self) -> dict[DockerLabelKey, str]:
         """returns a dictionary of strings as required by docker"""
         return {
             to_simcore_runtime_docker_label_key(k): f"{v}"
-            for k, v in sorted(self.model_dump().items())
+            for k, v in sorted(self.model_dump(warnings=False).items())
```

Comment:
really? what were the warnings about?

Reply:
The problem here is that the `self` object is created using `.model_construct` and the `node_id` was a `str` instead of `UUID`.

Reply:
but then silencing the warning is maybe not the right way? or am I missing something? please ping me to explain